src/DigitalSubscriberAppBanner.stories.tsx

```diff
@@ -1,26 +1,29 @@
 import React, { ReactElement } from 'react';
 import { withKnobs, text } from '@storybook/addon-knobs';
-import { DigitalSubscriberAppBanner } from './DigitalSubscriberAppBanner';
+import { BrazeMessage } from './BrazeMessage';
 import { StorybookWrapper } from './utils/StorybookWrapper';
 
 export default {
-    component: DigitalSubscriberAppBanner,
+    component: 'DigitalSubscriberAppBanner',
     title: 'Components/DigitalSubscriberAppBanner',
     decorators: [withKnobs],
 };
 
 export const defaultStory = (): ReactElement => {
     return (
         <StorybookWrapper>
-            <DigitalSubscriberAppBanner
+            <BrazeMessage
+                componentName={text('componentName', 'DigitalSubscriberAppBanner')}
```

Comment:
It'd be magic if this was a drop-down. No harm for now though, this will only matter when there are multiple options.

Reply:
Yep good point. I think okay to leave for a future version though like you suggested. This did remind me to add `slotName` as an (unused) prop control in Storybook, so the fields line up with what's in Braze.

Reply:
OK yeah, I was gonna chase that up with you separate, but great to do it at the same time for doing a single release, I guess.